### PR TITLE
Add recommended Ethernet adapter for the Wii mini

### PR DESCRIPTION
**Pull request by Devnol**

### CODE REVIEW

**Review by KcrPL — CHANGES_REQUESTED** [review]
Hmm... I don't think we should put links to Amazon etc. on wii.guide.

**Comment by Devnol** [comment]
This edit was requested by the Wii mini community as a lot of people have been asking for an Ethernet adapter compatible with the console that is still actively being sold. The links as you can see are non-affiliate but honestly it's up to you.

**Comment by KcrPL** [comment]
Actually that would be really useful now that I think about it. Let's wait for feedback from someone else.

**Comment by Peter0x44** [comment]
I don't like the link either, but a large part is wording... I don't think there are much better alternatives
How about "this ethernet adapter has been tested and is known to work" or so, I don't like "we recommend"

**Comment by Devnol** [comment]
Fair enough,

**Comment by Artuto** [comment]
~~Let's put an affiliate link 🤑🥴~~

**Comment by Devnol** [comment]
I don't see any reason not to honestly

**Comment by larsenv** [comment]
I saw this earlier but didn't respond.

@KcrPL @Peter0x44 You're not really providing much of a reason why you're opposed to linking to Amazon. I personally think it's perfectly fine, no one will care as everyone knows about Amazon.

I say merge it, and perhaps we could also put some other Amazon links when we find something necessary (e.g. linking to a Wii Speak, recommended SD Card, USB drive, etc.)

**Review by larsenv — CHANGES_REQUESTED** [review]
"If you're planning to buy an adapter, first search with your favorite search engine or look at the product page and confirm if it uses this specific chip. This adapter from UGreen has been tested by many community members and is known to work:" is too wordy.

Simply say that the UGREEN adapter is recommended.

**Comment by Devnol** [comment]
Yeah I guess it is a little to verbose but the part where we mention what chip the adapter needs to have is sort of crucial. I will trim the period tomorrow. Thanks for the feedback guys and have a good day.

**Comment by Peter0x44** [comment]
> You're not really providing much of a reason why you're opposed to linking to Amazon
I never really said I was opposed
> Simply say that the UGREEN adapter is recommended.
I don't think we should say explicitly "recommended" as that puts some burden of responsibility on us that it will actually work, which we can't guarantee

**Comment by Devnol** [comment]
TBH I totally forgot about this, sorry for being late but here you go, the changes you requested.

**Comment by Peter0x44** [comment]
Much better, I think it's good now

**Review by Peter0x44 — APPROVED** [review]

File: _pages/en_US/cios-mini.md [hunk 1]
```diff
@@ -46,6 +46,9 @@ Take a note of the version number under notes (`IOS57-64-` ending in either `v31
 ##### Enabling Ethernet
 If you wish to use Wiimmfi with Ethernet on a Wii mini, you have to run the [Ethernet Enabler Homebrew](/assets/files/Wii_Mini_Ethernet_Enable.zip) app made by Fullmetal5. To run it, just unzip it in the `apps` folder of your USB drive and run it from the Homebrew Channel.
 
+The Wii and Wii mini consoles are only known to work with Ethernet adapters using the AX88772 chip. Please check that the product is compatible before you buy. This one from UGREEN is known to be compatible: [amazon europe](https://www.amazon.de/dp/B00MYT481C), [amazon us](https://www.amazon.com/dp/B08DRKYKMM/)
```
Comment (Rexogamer) [inline _pages/en_US/cios-mini.md:49]: minor capitalisation nitpick:
```suggestion
The Wii and Wii mini consoles are only known to work with Ethernet adapters using the AX88772 chip. Please check that the product is compatible before you buy. This one from UGREEN is known to be compatible: [Amazon Europe](https://www.amazon.de/dp/B00MYT481C), [Amazon US](https://www.amazon.com/dp/B08DRKYKMM/)
```

Comment (Devnol) [reply _pages/en_US/cios-mini.md:49]: image
amazon is marketed lowercase

Comment (Devnol) [reply _pages/en_US/cios-mini.md:49]: elsewhere it's Amazon but whatever I guess I can merge it

**Review by Rexogamer — APPROVED** [review]

**Comment by Peter0x44** [comment]
No one else has voiced any objections over how this is worded, and there has been more than enough time, so I'm going to merge it.
